docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -858,6 +860,21 @@ By default, the users' organization and role is reset on every new login.
 > With Grafana 10, if `oauth_skip_org_role_update_sync` option is set to `false`, users with no mapping will be
 > reset to the default organization role on every login. [See `auto_assign_org_role` option]({{< relref ".#auto_assign_org_role" >}}).
 
+### [auth].<provider> skip_org_role_update_sync
+
+If you want to prevent org roles from being synced for a specific OAuth integration, set `skip_org_role_update_sync` to `true`.
+
+The `oauth_skip_org_role_update_sync` are independent of each other. `skip_org_role_sync` is specific to the provider integration, while `oauth_skip_org_role_sync_update` is a generic setting that only applies to `Github`, `Gitlab`, `AzureAD`, `Generic OAuth` integrations. We are deprecating the generic setting in favor of the provider specific settings.
```

Comment:
> The `oauth_skip_org_role_update_sync` are independent of each other

Can't understand what is meant by this statement. `The oauth_skip_org_role_update_sync is independent  from a specific providers' skip_org_role_update_sync`?

Reply:
Yea, they are not independant. Poor choice of words, I will reframe it more like separate settings